### PR TITLE
Hardcode times for HTN

### PR DESCRIPTION
**Pull request by BhavikaSharma**
Hardcode start time and end time for HTN and switch text between time until hacking starts and time until hacking ends.

I kept in commented lines to easily revert later on to a full countdown but I can change that if we want. 

image

### CODE REVIEW

File: blimp-live-widgets/src/components/Countdown/index.js [hunk 1]
```diff
@@ -5,27 +5,72 @@ import styles from './styles.css'
 
 export default class Countdown extends Component {
   constructor(props: Props) {
-    super(props);
+    var currentTime = new Date()
+    var htnStartTime = new Date(2019, 8, 13, 23, 30, 0, 0) // real HTN start time
+    // var htnStartTime = new Date(2019, 8, 7, 18, 0, 0, 0) // testing
```
Comment (jj8huang) [inline blimp-live-widgets/src/components/Countdown/index.js]: We can remove this comment I think.

File: blimp-live-widgets/src/components/Countdown/index.js [hunk 2]
```diff
@@ -5,27 +5,72 @@ import styles from './styles.css'
...
     if (this.state.timeRemaining > 0) {
-      this.state.timeRemaining = this.state.timeRemaining - 1;
-      this.formatTime();
+      this.state.timeRemaining = this.state.timeRemaining - 1
+      this.formatTime()
+    } else if (!this.state.htnHackingStarted) {
+      var htnStartTime = new Date(2019, 8, 13, 23, 30, 0, 0) // real HTN start time
+      var htnStartEnd = new Date(2019, 8, 15, 9, 0, 0, 0) // real HTN end time
+
+      // var htnStartTime = new Date(2019, 8, 7, 18, 0, 0, 0) // testing
```
Comment (jj8huang) [inline blimp-live-widgets/src/components/Countdown/index.js]: Remove comment

File: blimp-live-widgets/src/components/Countdown/index.js [hunk 3]
```diff
@@ -5,27 +5,72 @@ import styles from './styles.css'
...
+      // var htnStartEnd = new Date(2019, 8, 7, 18, 1, 0, 0) // testing
```
Comment (jj8huang) [inline blimp-live-widgets/src/components/Countdown/index.js]: Remove comment

File: blimp-live-widgets/src/components/Countdown/index.js [hunk 4]
```diff
@@ -55,41 +100,48 @@ export default class Countdown extends Component {
...
+        </div>
+        <div>
+          <input className={styles.digit} type='text' value={displayHr} onClick={this.handleFocus} onChange={this.changeHours} />
+          <span className={styles.colon}>:</span>
+          <input className={styles.digit} type='text' value={displayMin} onClick={this.handleFocus} onChange={this.changeMinutes} />
+          <span className={styles.colon}>:</span>
+          <input className={styles.digit} size='1' type='text' value={displaySec} onClick={this.handleFocus} onChange={this.changeSeconds} />
+          <br />
+        </div>
+        {/* <button className={styles.button} onClick={this.startCountdown}> Start </button>
```
Comment (jj8huang) [inline blimp-live-widgets/src/components/Countdown/index.js]: Is this just so we can reuse these again later? If yes that's fine, if not we should remove.

Comment (BhavikaSharma) [reply blimp-live-widgets/src/components/Countdown/index.js]: Yep - to reuse later.

**Pull request merged**